zerver/webhooks/statuspage/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications for your Statuspage.io subscriptions!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+If you're a company and you'd like to enable webhook notifications,
+go to the **Subscriber's Settings** in **Notifications** tab and enable
```

Comment:
Replace `**Subscriber's Settings**` with just `**Settings**`, your screenshot doesn't say `Subscriber's Settings`, it says only `Settings`